blocks/ai-text/package.json

```
@@ -54,7 +54,7 @@
     "blockType": {
       "entryPoint": "react"
     },
-    "name": "@hash/ai-text",
+    "name": "@luisbett/ai-text",
```

I think this diff was unintentional, right?
```suggestion
    "name": "@hash/ai-text",
```

---

Yup, my bad!